pkgs/applications/office/planner/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchurl, meson, ninja, cmake, pkg-config,
+  desktop-file-utils, shared-mime-info, makeWrapper,
+  glib, gtk3, libxml2, libxslt, libgda, gsettings-desktop-schemas }:
+
+stdenv.mkDerivation rec {
+  pname = "planner";
+  version = "0.14.91";
+
+  src = fetchurl {
+    url = "https://gitlab.gnome.org/World/${pname}/-/archive/${version}/${pname}-${version}.tar.bz2";
+    sha256 = "1hd2adm6sg0dqwv32ykw4xv3q7x4hhjckxcx8p55amvvsy3jacy5";
+  };
+
+  postPatch = ''patchShebangs \
+                  meson_post_install.sh \
+                  tools/strip_trailing_white_space.sh \
+                  tests/python/task-test.py'';  # grep '^#! */' -R .
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ meson ninja cmake pkg-config
+                        desktop-file-utils shared-mime-info
+                        makeWrapper ];
+  buildInputs = [ glib gtk3 libxml2 libxslt libgda
+                  gsettings-desktop-schemas ];
+
```

Comment:
```suggestion
  nativeBuildInputs = [ 
    desktop-file-utils
    cmake
    makeWrapper
    meson
    ninja
    pkg-config
    shared-mime-info
  ];

  buildInputs = [ 
    libgda
    libxml2 
    libxslt
    glib
    gsettings-desktop-schemas
    gtk3 
  ];

```

Reply:
Why both cmake and meson?

Reply:
I don't remember: but yes it's not needed.